src/lib/ui/DeviceDetails/DeviceDetails.tsx

```diff
@@ -54,6 +61,39 @@ function LockDeviceDetails(props: { device: LockDevice; onBack?: () => void }) {
   const accessCodeLength =
     device.properties?.schlage_metadata?.access_code_length
 
+  function generateDeviceAlerts() {
```

Comment:
Not sure if we want to move this somewhere like `lib/utils`. Let me know!

Reply:
I'd only move it if it would be used in another component. I suppose that's premature  simplification, though—I'll leave it as-is for now.